sites/partners/src/components/listings/PaperListingForm/sections/RankingsAndResults.tsx

```diff
@@ -211,6 +213,7 @@ const RankingsAndResults = ({ listing, isAdmin }: RankingsAndResultsProps) => {
                   name={"lotteryEndTime"}
                   id={"lotteryEndTime"}
                   register={register}
+                  setValue={setValue}
```

Comment:
Previously if these fields were blank on publish/save, they would all be highlighted red with a warning. Now if they are left blank there is no red warning but an error still appears at the top. Not sure why only the date field is still correctly showing that error

Reply:
@mcgarrye Can you take one more look at this? I'm not entirely sure why it previously worked in main, but as those date & time fields are required, I wouldn't expect them to validate entry on save without there being a `required` prop. So I've added those and it seems to be acting OK on the PR to me now.

Reply:
![Screenshot 2024-10-18 at 12 23 56 PM](https://github.com/user-attachments/assets/9f3c39e7-1a93-41d6-979c-7e8b8b0a081d)
I'm still not seeing the red highlight on the fields :/

Reply:
@mcgarrye Ah, so it _is_ working on the date field, but not on the time fields if one of those is blank. Hmm, I will investigate!